src/display/api.js

```diff
@@ -897,76 +885,73 @@ class PDFDocumentProxy {
  */
 
 /**
  * Page render parameters.
  *
  * @typedef {Object} RenderParameters
  * @property {Object} canvasContext - A 2D context of a DOM Canvas object.
- * @property {PageViewport} viewport - Rendering viewport obtained by
- *                          calling the `PDFPageProxy.getViewport` method.
- * @property {string} [intent] - Rendering intent, can be 'display' or 'print'
- *                    (default value is 'display').
- * @property {boolean} [enableWebGL] - Enables WebGL accelerated rendering
- *                     for some operations. The default value is `false`.
- * @property {boolean} [renderInteractiveForms] - Whether or not
- *                     interactive form elements are rendered in the display
- *                     layer. If so, we do not render them on canvas as well.
- * @property {Array<any>} [transform] - Additional transform, applied
- *                        just before viewport transform.
- * @property {Object} [imageLayer] - An object that has beginLayout,
- *                    endLayout and appendImage functions.
+ * @property {PageViewport} viewport - Rendering viewport obtained by calling
+ *   the `PDFPageProxy.getViewport` method.
+ * @property {string} [intent] - Rendering intent, can be 'display' or 'print'.
+ *   The default value is 'display'.
+ * @property {boolean} [enableWebGL] - Enables WebGL accelerated rendering for
+ *   some operations. The default value is `false`.
+ * @property {boolean} [renderInteractiveForms] - Whether or not interactive
+ *   form elements are rendered in the display layer. If so, we do not render
+ *   them on the canvas as well.
+ * @property {Array<any>} [transform] - Additional transform, applied just
+ *   before viewport transform.
+ * @property {Object} [imageLayer] - An object that has `beginLayout`,
+ *   `endLayout` and `appendImage` functions.
  * @property {Object} [canvasFactory] - The factory instance that will be used
- *                    when creating canvases. The default value is
- *                    {new DOMCanvasFactory()}.
- * @property {Object} [background] - Background to use for the canvas.
- *                    Can use any valid canvas.fillStyle: A DOMString parsed as
- *                    CSS <color> value, a CanvasGradient object (a linear or
- *                    radial gradient) or a CanvasPattern object (a repetitive
- *                    image). The default value is 'rgb(255,255,255)'.
+ *   when creating canvases. The default value is {new DOMCanvasFactory()}.
+ * @property {Object} [background] - Background to use for the canvas. Can use
+ *   any valid `canvas.fillStyle`: a `DOMString` parsed as CSS <color> value,
+ *   a `CanvasGradient` object (a linear or radial gradient) or a
+ *   `CanvasPattern` object (a repetitive image). The default value is
+ *   'rgb(255,255,255)'.
  * @property {Object} [annotationStorage] - Storage for annotation data in
```

Comment:
Shouldn't this be `@property {AnnotationStorage} [annotationStorage] - ...`, since this shouldn't just a regular Object?